xgcm/grid.py

```diff
@@ -758,6 +765,97 @@ def max(
             keep_coords,
         )
 
+    def transform(
+        self,
+        da,
+        target,
+        target_data=None,
+        target_dim=None,
+        method="linear",
+        mask_edges=False,
+    ):
```

Comment:
Where is the docstring for `transform`. It needs to be fully filled out!

Reply:
Agreed. Let me fix the mess I made with the CI and then fix this.

Reply:
What do you think about the docstrings I added?